test/test-run-appmetrics-usage.js

```diff
@@ -0,0 +1,51 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: strong-supervisor
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var debug = require('./debug');
+var run = require('./run-with-ctl-channel');
+var tap = require('tap');
+
+tap.test('appmetrics are forwarded via parentCtl', function(t) {
```

Comment:
I think it's the `test-runctl-clusterctl.js` test that's timing out. It also has its own custom timeout value, which might be part of it.

I'll tweak it and see if it changes anything.